Chap_API_Init.tex

```diff
@@ -372,8 +311,12 @@ \subsection{\code{PMIx_tool_connect_to_server}}
 %%%%
 \descr
 
-Switch connection from the current \ac{PMIx} server to another one, or initialize a connection to a specified server. Closes the connection, if existing, to a server and establishes a connection to the specified server. This function can be called at any time by a \ac{PMIx} tool to shift connections between servers.
-The process identifier assigned to this tool is returned in the provided \refstruct{pmix_proc_t} struct. Passing a value of \code{NULL} for this parameter is allowed if the user wishes solely to connect to the \ac{PMIx} server and does not require return of the identifier at that time.
+A tool may call \refapi{PMIx_tool_init} with the \refattr{PMIX_TOOL_DO_NOT_CONNECT} attribute in which case they can use this function to connect to a specific server.
+Additionally, a tool may use this function to switch connection from the current \ac{PMIx} server to another one
```

Comment:
Seems like something is missing at end of this sentence, leading to next line.

Reply:
Humm. Yeah, I think I missed something here. I'll fix and push in a minute. Thanks!

Reply:
Fixed in latest push